QuasarHelios2015/src/org/team1540/quasarhelios/DisplacementSensor.java

```diff
@@ -0,0 +1,13 @@
+package org.team1540.quasarhelios;
+
+import java.awt.geom.Point2D;
```

Comment:
Please don't use AWT unless you're doing graphics in your code somewhere. I'd rather you write your own Point2D than depend on functionality from Java's graphics library.